### DISCUSSION TITLE
Creation of a new helmrelease causes a panic on the helm controller: runtime error: invalid memory address or nil pointer dereference

### DISCUSSION THREAD

**Discussion by soumyabk** [fluxcd/flux2#3942] [Q&A]
version of helm controller: v0.31.2
we have the --feature-gates=DetectDrift=true enabled and seems like when new helm-releases are created it causes a panic. Disabling the flag and creating the helmrelease seems to resolve it. 
Is there a fix/work-around for this?

**Comment by stefanprodan** [comment]
This was fixed in https://github.com/fluxcd/helm-controller/pull/672
